release-please-config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"bump-minor-pre-major": true,
+	"bump-patch-for-minor-pre-major": true,
+	"draft": true,
```

Comment:
What does this option?
Is it for the pull request or release?

Reply:
I thought it was for PR but apearantly, it was for release. I will make this false.